File: Point Adjust Robot/Core/DesignPatterns/Command/CommandCover.cs
using Newtonsoft.Json;
using Point_Adjust_Robot.Core.Interface;
using Point_Adjust_Robot.Core.Model;

namespace Point_Adjust_Robot.Core.DesignPatterns.Command
{
    public class CommandCover : CommandFront
    {
        public List<WorkShiftCover> coverWorkShifts { get; set; } = new List<WorkShiftCover>();

    }
}

File: FrontApp/src/services/commands/CommandCover.js
import CoverWorkshift from "../../model/CoverWorkshift";
import Command from "./Command";

export default class CommandCover extends Command {
  constructor(coverWorkShifts = Array) {
    super(coverWorkShifts);
    this.coverWorkShifts = this.data.map(d => new CoverWorkshift(d));
    delete this.data
  }
  Execute() {}
}
